### PR TITLE
Ability to provide value for each transaction.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

**Comment by d11e9** [comment]
Kinda overlaps with https://github.com/d11e9/browser-solidity/tree/accounts branch I've got in progress. But short term probably the right way the go.

File: assets/js/universal-dapp.js [hunk 1]
```diff
@@ -388,7 +413,7 @@ UniversalDApp.prototype.runTx = function( data, args, cb) {
     var isConstructor = args.bytecode !== undefined;
 
     var gas = self.options.getGas ? self.options.getGas : 1000000;
-    var value = self.options.getValue ? self.options.getValue : 0;
+    var value = (args.value || (self.options.getValue ? self.options.getValue : 0)) - 0;
```
Comment (d11e9) [inline assets/js/universal-dapp.js:416]: With regards to the branch I mentioned, The idea was to have a single tx input area (same place as origin) and using the `options.getValue` handler to get the value desired by the user for the current tx.

**Comment by chriseth** [comment]
Oh sorry, I was not aware of your work.
I just hacked together this PR for a demo, so we can also just close it if you have something better in mind.
In the end, it also did not work. I'm not sure if it works better for your, but I had problems with contract creation transactions not being processed properly with a geth backend node (might be due to some gas problems).

**Comment by axic** [comment]
Any updates on this? Would be nice to have merged. Would complement #126 and #131.

**Comment by chriseth** [comment]
I abandoned it because I thought @d11e9 would have a better solution :-)
The UI also does not really please me.

**Pull request closed without merging**